.github/workflows/merlin.yml

```diff
@@ -137,13 +137,6 @@ jobs:
           E2E_USE_GOOGLE_OAUTH: false
         working-directory: ./python/sdk
         run: make unit-test
-      - uses: codecov/codecov-action@v4
```

Comment:
Ah it's because the test coverage scan is inaccurate. Sometimes even without making any code changes, the code coverage report reports a massive % drop and fails the CICD job :/